docs/further_useful_features.rst

```diff
@@ -0,0 +1,459 @@
+.. _further_useful_features:
+
+=================================
+Further useful kartothek features
+=================================
+
+This document introduces additional features of ``kartothek`` that allow users to
+partition and index their data during writing, update already written data and
+keep their data stores 'clean' with garbage collection.
+
+The intent of this document is to expose users to some additional features of ``kartothek``
+in a bid to encourage deeper exploration of ``kartothek`` beyond 'getting started' and help
+better enable developers to start thinking about how it can (or cannot) be useful in their
+particular applications.
+
+.. _storage_specification:
+
+Storage specification
```

Comment:
This section is not _further reading_ but a *hard* requirement for using kartothek. Please move this at the top of the getting started guide. At the very least the getting started guide should highlight the most important parts (e.g. h prefix

Reply:
Highlighted h prefix stores